code/modules/spells/spell_types/rod_form.dm

```diff
@@ -4,14 +4,15 @@
 	clothes_req = 1
 	human_req = 0
 	charge_max = 250
-	cooldown_min = 100
+	cooldown_min = 200
```

Comment:
Now this bit right here makes sense. Tweaking the charge time is sensible. Raising cost from 3 to inaccessible 40 is not.